instrumentation/http/src/main/java/brave/http/HttpServerHandler.java

```diff
@@ -61,14 +75,36 @@
     );
     this.tracer = httpTracing.tracing().tracer();
     this.sampler = httpTracing.serverSampler();
+    this.defaultExtractor = httpTracing.tracing().propagation().extractor(HttpServerRequest.GETTER);
+    this.defaultHandler = adapter == HttpServerAdapter.DEFAULT
```

Comment:
this is tricky games so we can keep portable with old code, yet use the new `public Span handleReceive(HttpServerRequest request)` method

Reply:
Not sure I understand the game but should adapter be passed in on the line below? Confused why we can ignore the adapter if it's not default